templates/zerver/emails/email.css

```diff
@@ -84,65 +80,52 @@ table td {
     margin-top: 20px;
 }
 
-.footer td,
-.footer span {
+.content-block,
+.apple-link {
```

Comment:
I renamed apple-link to physical-address and `content-block` to `footer-content-block` to make this more readable.